po/ru.po

```diff
@@ -1230,13 +1230,13 @@ msgid "Resolution:"
 msgstr "Разрешение:"
 
 msgid "Resources"
-msgstr ""
+msgstr "Ресурсы"
 
 msgid "Restart\\Restart the mission from the beginning"
 msgstr "Заново\\Начать данную миссию с начала"
 
 msgid "Restoring CBot execution state"
-msgstr ""
+msgstr "Восстановление игры"
```

Comment:
> Maybe we need to improve only the string:
> msgid "Restoring CBot execution state"
> in Russian it's translated in this poll as "Restoring game" I think there should be more correct translation.

~ **rvasin** in https://colobot.info/forum/showthread.php?tid=863&pid=7593#pid7593